kusto/queryopts.go

```diff
@@ -128,10 +128,10 @@ func NoTruncation() QueryOption {
 	}
 }
 
-// ResultsProgressiveDisable disables the progressive query stream.
-func ResultsProgressiveDisable() QueryOption {
+// ResultsProgressiveEnabled enables the progressive query stream.
+func ResultsProgressiveEnabled() QueryOption {
```

Comment:
nit: EnableProgressiveResults() and EnableProgressiveResultsValue = "enable_progressive_results"